blocks/product-assortment-carousel-block/README.md

```diff
@@ -26,14 +26,16 @@ The block relies on Algolia data to render the products in the carousel. The fie
 - Image
 - Attributes - array of objects
   - `name = product_url`
-- Prices - array of object
-  - `type = List`
-  - `type = Sale`
-  - For each price
+- Pricing - array of object
+  - For each pricing object
     - `currencyLocale`
     - `currencyCode`
-    - `amount`
-    - `type`
+    - `currencyDisplayFormat`
+    - Prices - array of object
+      - For each price object
+        - `type` - `Sale` or `List`
+        - `amount`
+        - `id`
```

Comment:
The code doesn't require the `id` field - and if we are not using it we should not document it as needed

Reply:
updated thanks